src/plugins/intel_cpu/src/nodes/llm_mlp.cpp

```diff
@@ -23,6 +23,8 @@ namespace intel_cpu {
 namespace node {
 
 #if defined(OPENVINO_ARCH_X86_64)
+static ReduceAdd2bh jit_reduce2bh_1(false);
```

Comment:
1. We should avoid global static variables al long as it might break OV static linkage into target application due to unspecified initialization order.
2. Static variables are not deallocated during CompiledModel dtor. It is recommended to reduce live scope of such objects. If we want to share kernels between different MLP nodes, than default CPU plugin caching mechanism should be used.

Reply:
Sure, changed back to local static (singleton).